### PR TITLE
Preparations for Elastic Beanstalk

### PR DESCRIPTION
**Pull request by farski**

### CODE REVIEW

**Comment by farski** [comment]
Running here for testing: http://prx-upload-beta.elasticbeanstalk.com

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+# Set correct environment variables.
+ENV HOME /root
+ENV RAILS_ENV production
+
+# Use baseimage-docker's init process.
+CMD ["/sbin/my_init"]
+
+# Expose Nginx HTTP service
+EXPOSE 80
```
Comment (kookster) [inline Dockerfile:13]: should this be 443 too? @chrisrhoden @farski seems like we should be using ssl for this, yes?

Comment (farski) [inline Dockerfile:13]: Can't use HTTPS on EB until a custom domain is set up, which is on my list for today.

Comment (kookster) [inline Dockerfile:13]: sure, premature critique, sorry.

Comment (cqr) [inline Dockerfile:13]: I think the typical arrangement is to have ELB do SSL termination

Comment (kookster) [inline Dockerfile:13]: It isn't clear to me that you can do it that way with elastic beanstalk.
This is the guide I remember - 
https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/SSLDocker.SingleInstance.html

Comment (cqr) [inline Dockerfile:13]: If there's a way to set up Elastic Beanstalk without an ELB, then I'm all for doing so in this case. For reasons that are probably obvious, the minute you install a load balancer you need to put the keys onto it, and if we're using VPC there's not really a good reason to do the encryption twice. Elastic Beanstalk absolutely supports this case and recommends it http://docs.aws.amazon.com/elasticbeanstalk/latest/dg/configuring-https.html

Comment (farski) [inline Dockerfile:13]: I haven't added LBs for any of the environments I've created so far.

Comment (cqr) [inline Dockerfile:13]: Yep, this was based on my misunderstanding that Elastic Beanstalk always set up an ELB.

**Pull request closed without merging**